### PR TITLE
ci: Add cache for cloud hypervisor

### LINKED ISSUE #2120: create cache for cloud hypervisor

### PR DESCRIPTION
**Pull request by GabyCT**
This adds the installation for cached cloud hypervisor.

Fixes #2120

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/ci_cache_components.sh [hunk 1]
```diff
@@ -156,6 +164,7 @@ Description: This script builds the cache of several Kata components.
 Options:
 
 -a      Run qemu experimental cache
+-c	Run cloud hypervisor cache
```
Comment (chavafg) [inline .ci/ci_cache_components.sh]: nit: indentation issue.

File: .ci/ci_cache_components.sh [hunk 2]
```diff
@@ -55,6 +55,13 @@ cache_kernel_artifacts() {
 	done
 }
 
+# This builds cloud hypervisor
+cache_cloud_hypervisor() {
+	local cloud_hypervisor_binary="/usr/bin/cloud-hypervisor"
+	local current_cloud_hypervisor_version=$(get_version "assets.hypervisor.cloud_hypervisor.url")
```
Comment (chavafg) [inline .ci/ci_cache_components.sh]: `s/url/version/`?

**Comment by GabyCT** [comment]
@chavafg changes applied

**Pull request merged**